components/homepage/HomepageNav.js

```diff
@@ -52,19 +52,39 @@ const HomepageNav = ({
     const regex = /\b\w+/;
     if (!regex.test(e.target.value) && e.target.value !== '') return;
     setSearch(e.target.value);
-    router.push({
+    /*router.push({
       pathname: router.pathname,
       query: { _q: encodeURI(e.target.value) },
```

Comment:
Clean up, if commented out code would not be used in the future

Reply:
solved

Reply:
Cool